packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -144,6 +144,8 @@ export class SidebarComponent extends BaseContainer {
 			[ elementHandle ]
 		);
 
+		await elementHandle.waitForElementState( 'stable' );
```

Comment:
To be frank, I suspect this part isn't necessary since the scroll action != animation and [according to the docs](https://playwright.dev/docs/actionability#stable) the `stable` state fires when an element `...maintained the same bounding box for at least two consecutive animation frames.`

Reply:
🤔  Do we still want to add it if it could be doing nothing? I'll let you make that decision

Reply:
I needed to verify if this was necessary, which I will do before merging this PR. Plan is to set up another branch, one without this change and make it run tests several times. I suspect it isn't necessary, but I've been wrong on these `waitForElementState` before.